Apps/frontend/src/components/DistributeOffsetButton.svelte

```diff
@@ -0,0 +1,23 @@
+<script>
```

Comment:
Why is a previous version of the DistributeButton merged in here? Why not wait for the merge into dev?

Reply:
Old merge from before christmas. Not used

Reply:
Deleted